### PR TITLE
Fix for stx-rosetta.Dockerfile

### PR DESCRIPTION
**Pull request by aravindgee**
- Use new repos: hirosystems/stacks-blockchain-api and stacks-network/stacks-blockchain
- Read Toml from stacks-network / stacks-blockchain/testnet/stacks-node/conf/

These updates will now let the docker image build and run, but the settings in the Toml need to be validated.

[Related Issue](https://github.com/hirosystems/stacks-blockchain-api/issues/1362)

### CODE REVIEW

File: docker/stx-rosetta.Dockerfile [hunk 1]
```diff
@@ -1,9 +1,9 @@
 ARG STACKS_API_VERSION=v0.71.2
```
[the remark this answers is not in the document]
Comment (aravindgee) [reply docker/stx-rosetta.Dockerfile]: Got it, done.

**Review by zone117x — COMMENTED** [review]
Have we tested this docker image to confirm it works? If so, LGTM. Although, I'm deferring to @rafaelcr for review of the env var changes.

**Comment by rafaelcr** [comment]
@aravindgee could you rebase/merge this branch with the latest PRs from master? We can see if it works once the CI runs

**Comment by aravindgee** [comment]
@rafaelcr Rebased & committed.

**Comment by aravindgee** [comment]
Closing  - duplicate of #1479

**Pull request closed without merging**